src/Goerli-DssSpell.sol

```diff
@@ -51,41 +47,92 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
     //
     // A table of rates can be found at
-    //    https://ipfs.io/ipfs/QmX2QMoM1SZq2XMoTbMak8pZP86Y2icpgPAKDjQg4r4YHn
+    //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
 
+    // --- Rates ---
+    uint256 constant ZERO_PCT_RATE             = 1000000000000000000000000000;
+    uint256 constant ZERO_ZERO_TWO_PCT_RATE    = 1000000000006341324285480111;
+    uint256 constant ZERO_ZERO_SIX_PCT_RATE    = 1000000000019020169709960675;
+    uint256 constant TWO_TWO_FIVE_PCT_RATE     = 1000000000705562181084137268;
+    uint256 constant THREE_SEVEN_FIVE_PCT_RATE = 1000000001167363430498603315;
+
     function officeHours() public override returns (bool) {
         return false;
     }
 
     function actions() public override {
-        wipeFromRWA009Urn();
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        onboardNewCollaterals();
-        drawFromRWA009Urn();
-
-        // Add RWA_TOKEN_FAB to changelog
-        DssExecLib.setChangelogAddress("RWA_TOKEN_FAB", RWA_TOKEN_FAB);
-
-        DssExecLib.setChangelogVersion("1.13.3");
-    }
+        // onboardNewCollaterals();
 
-    function wipeFromRWA009Urn() internal {
-        // wipe DAI
-        RwaUrnLike(RWA009_A_URN_OLD).wipe(RWA009_DRAW_AMOUNT);
-
-        // free old RWA009 Token from the URN
-        RwaUrnLike(RWA009_A_URN_OLD).free(1 * WAD);
-    }
+        // ----------------------------- RWA Draws -----------------------------
+        // https://vote.makerdao.com/polling/QmQMDasC#poll-detail
+        // Weekly Draw for HVB
+        address RWA009_A_URN = DssExecLib.getChangelogAddress("RWA009_A_URN");
+        // Draw once to catch up to mainnet
+        RwaUrnLike(RWA009_A_URN).draw(RWA009_DRAW_AMOUNT);
```

Comment:
We should probably clarify that we are only doing this for next week because there was no spell for Goerli in the current week.